### PR TITLE
Polkadot JS Extension Moonbeam Address Support Tutorial

### PR DESCRIPTION
**Pull request by themacexpert**
Adds the "Interacting with Moonbeam Using the Polkadot.js Extension" Page under Tokens including the following:
- Intro, how the Extension works by injecting accounts.
- How to Install the Extension
- Update Metadata
- Creating a Moonbeam Address
- Interacting with Polkadot.js Apps

### CODE REVIEW

File: tokens/connect/polkadotjsextension.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+On this final page of the account creation wizard, take the following steps:
+
+ 1. Select the `Moonriver` Network. If you create an account in another format, such as Polkadot or Kusama, you'll be unable to later convert that to the Ethereum-style address format used in Moonriver and you'll need to create a new Moonriver address 
+ 2. Give this account a descriptive name
+ 3. Set a password, this will be used when signing transactions
+ 4. Confirm the password
+ 5. Click “Add the account with the generated seed” to complete the account generation 
+
+![Finish Account Creation](/images/tokens/connect/polkadotjsext/polkadotjs-ext-5.png)
```
Comment (albertov19) [inline tokens/connect/polkadotjsextension.md:73]: This image is too big. In such cases you can use HTML image imports. Check Nifty Wallet for example

```
<img src="/images/tokens/connect/nifty/nifty-1.png" alt="Set wallet password" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
```

**Review by albertov19 — CHANGES_REQUESTED** [review]
You need to add a logo in `/moonbeam-mkdocs/moonbeam-docs/images/index-pages/tokens/connect`

File: tokens/connect/polkadotjsextension.md (hunk 1 above)
Comment (themacexpert) [reply tokens/connect/polkadotjsextension.md]: Thank you for the feedback @albertov19, should be all set now 👍

**Comment by albertov19** [comment]
Converting it to draft for now

**Comment by themacexpert** [comment]
Closing this as Stale. We can re-open if needed.

**Pull request closed without merging**